tests/dummy/app/services/apollo.ts

```diff
@@ -1,17 +1,18 @@
 import ApolloService from 'ember-apollo-client/services/apollo';
-import { computed } from '@ember/object';
+import { computed } from '@ember-decorators/object';
 import {
   InMemoryCache,
   IntrospectionFragmentMatcher,
 } from 'apollo-cache-inmemory';
 import TypeIntrospectionQuery from 'dummy/utils/graphql-type-query';
 
-export default ApolloService.extend({
-  cache: computed(function() {
+export default class extends ApolloService {
+  @computed
```

Comment:
Probably doesn't need to be a computed.

Reply:
Same here. It's a computed to make it cache / lazily instantiated. Using an assignment would have the same effect, but is eagerly evaluated. Just removing the `@computed` and making this a regular getter would return a new `InMemoryChache` instance every time.